include/rmm/device_scalar.hpp

```diff
@@ -123,7 +141,7 @@ class device_scalar {
    * @param stream CUDA stream on which to perform the copy
    */
   template <typename Dummy = void>
-  auto set_value(T host_value, cudaStream_t stream = 0)
+  auto set_value(T const &host_value, cudaStream_t stream = 0)
     -> std::enable_if_t<std::is_fundamental<T>::value, Dummy>
```

Comment:
Let's rename `Dummy` => `Placeholder` or something, while we're here.

Reply:
Yes, good catch.